src/components/index/InformationBanner/InformationBanner.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styles from "./InformationBanner.module.scss";
+import informationBannerimage from "@/assets/information_banner_image.png";
+
+
+import Button from "@/components/global/Button";
+
+export default function InformationBanner() {
```

Comment:
I może InformationBannerText dać do innego komponentu? W sumie to się powtarza i tak byśmy uniknęli redundancji.